apps/dapp/src/components/Pages/Ascend/components/AuctionContext/index.tsx

```diff
@@ -72,22 +72,22 @@ interface Props {
 
 export const AuctionContextProvider: FC<Props> = ({ pool, children }) => {
   const { wallet } = useWallet();
-  const [buy, sell] = pool.tokens;
+  const [sell, buy] = pool.tokens;
```

Comment:
is this due to the ordering of tokens in goerli? Are the mainnet tokens going to be in the same alphanumeric order?

Reply:
good point, ill fix this.

Reply:
i've been assuming that the tokens would be sorted from the graph but probably not reasonable to assume. updating that list as well